packages/@okta/vuepress-site/docs/reference/rl-previous/index.md

```diff
@@ -32,4 +32,6 @@ The following are the high capacity rate limits per minute that apply across the
 | `/bc/image/fileStoreRecord`                                                | 3000      | 3000         |
 | `/bc/globalFileStoreRecord`                                                | 3000      | 3000         |
 
+>**Note:** The increased rate limit applies only to the exact endpoints in the previous table. For example, the high-capacity rate limit only applies to `api/v1/apps/${id}` and not to paths such as `api/v1/apps/${id}/users`.
```

Comment:
Not all of the endpoints listed above are exact match.   The following endpoints listed in the above table are not exact matches:

- /api/v1
- /api/v1/authn
- /bc/image/fileStoreRecord
- /bc/globalFileStoreRecord
- /api/v1/sessions
- /oauth2/{authorizationServerId}/v1
- /oauth2/v1